### PR TITLE
Fix graphana layout on pop-over

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
Fixes #166634073

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots

Current: 

image

Fix:

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/feinstaub-api.js [hunk 1]
```diff
@@ -3,6 +3,8 @@
 // const URL = 'https://api.airquality.codeforafrica.org/v2/sensors/?type=PPD42NS,HPM,PMS1003,PMS3003,PMS5003,PMS6003,PMS7003,SDS021,SDS011'
 const URL = "https://api.sensors.africa/v2/nodes/";
 
+//const URL = "http://staging.api.sensors.africa/v2/nodes/";
```
Comment (karimkawambwa) [inline src/feinstaub-api.js:6]: Would you be able to set this change to be an environment variable.
Then you can have 
```
.env with "https://api.sensors.africa/v2/nodes/"
``` 
and 
```
.env.development with "https://staging.api.sensors.africa/v2/nodes/"
```

Its not a must to be done in this PR

File: src/cell-info.vue [hunk 2]
```diff
@@ -29,15 +29,15 @@
...
 					a(:id="'graph_'+sensor.o.id+'_off'" class="graph_off" onclick="var sensor=this.id.substring(0,this.id.length-4);document.getElementById(sensor).style.display='none';document.getElementById(sensor+'_on').style.display='';document.getElementById(sensor+'_off').style.display='none'; document.getElementById('images_'+sensor.substr(6)).innerHTML=''; return false;" href='#' style='color:white; text-decoration: none; display: none;') (-)&nbsp; {{sensor.o.id}}
```
Comment (karimkawambwa) [inline src/cell-info.vue:33]: Oh gosh ummm, could you move this onclick string function to be a code function s'il vous plait?... @nerdlet

**Review by karimkawambwa — CHANGES_REQUESTED** [review]
Looks good but kindly see comments...

File: src/feinstaub-api.js (hunk 1 above)
Comment (nerdlet) [reply src/feinstaub-api.js:6]: @karimkawambwa I'll have to create a separate PR for this fixes

**Pull request merged**